src/sites/dMCDA.jl

```diff
@@ -8,6 +8,25 @@ using InteractiveUtils: subtypes
 using ADRIA: order_ranking, adria_vikor, adria_topsis
 
 
+jmcdm_ignore = [
+    JMcDM.CRITIC.CriticMethod,
+    JMcDM.COPRAS.CoprasMethod,
+    JMcDM.MOOSRA.MoosraMethod,
+    JMcDM.MEREC.MERECMethod,
+    JMcDM.ELECTRE.ElectreMethod,
+    JMcDM.PROMETHEE.PrometheeMethod,
+    JMcDM.Topsis.TopsisMethod,
```

Comment:
Rose, I've removed TOPSIS and VIKOR from consideration, but did you intend to keep these in for a reason I wasn't aware of?

Reply:
That probably makes sense to avoid overlap. I kept them in incase we wanted a "fast version" because they are quite a bit faster than the JMcDM implementations, but it's not very noticable when running a lot of simulations.

Reply:
>  it's not very noticable when running a lot of simulations.

That's when it should be very noticeable? I'm noting roughly an extra minute in run times compared to when we were only considering the three methods for 1024 scenarios.

I'll do a more in-depth profiling, but noting that seemingly small increases has a large implication for scenario/sensitivity analysis.